### PR TITLE
[script.myepisodes] 2.0.1

### PR DESCRIPTION
**Pull request by maximeh**
### Description
Update due to some changes introduced in Leia in the hookup of the player playback, it created a bug with remote content media server (SMB/NFS) which introduced an enough latency to trigged the problem (basically, the length of the video was always 0 because the player was ready but not the content).
### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.myepisodes/addon.xml [hunk 1]
```diff
@@ -14,8 +14,7 @@
     <description lang="en_GB">Automatically set watched episodes on MyEpisodes.com</description>
     <platform>all</platform>
     <language>en fr it pt es</language>
-    <forum></forum>
-    <website></website>
+    <website>http://github.com/maximeh/script.myepisodes</website>
```
Comment (Rechi) [inline script.myepisodes/addon.xml]: Website should list the site which provides the content or functionality.
```suggestion
    <website>myepisodes.com</website>
```

File: script.myepisodes/default.py [hunk 2]
```diff
@@ -124,7 +124,7 @@ def _addShow(self):
             added = 32925
         utils.notif("%s %s" % (self.title, _language(added)))
 
-    def onPlayBackStarted(self):
+    def onAVStarted(self):
```
Comment (Rechi) [inline script.myepisodes/default.py]: For Krypton you still have to listen for the `onPlayBackStarted` callback.

File: script.myepisodes/default.py [hunk 3]
```diff
@@ -124,7 +124,12 @@ def _addShow(self):
             added = 32925
         utils.notif("%s %s" % (self.title, _language(added)))
 
+    # For backward compatibility
     def onPlayBackStarted(self):
+        self.onAVStarted()
```
Comment (Rechi) [inline script.myepisodes/default.py]: As Leia still sends `onPlayBackStarted` callbacks, this call has to be limited for Krypton only.
Have a look at #977 and #981 how that can be implemented.

Comment (maximeh) [reply script.myepisodes/default.py]: Ah crap yeah ! Thanks for the tips ! I'll do that then.

File: script.myepisodes/default.py [hunk 4]
```diff
@@ -14,6 +14,7 @@
 from myepisodes import MyEpisodes
 
 _addon = xbmcaddon.Addon()
+_kodiversion = float(_addon.getAddonInfo('version')[0:4])
```
Comment (Rechi) [inline script.myepisodes/default.py]: This isn't the Kodi version, it's the version of your addon.
```suggestion
_kodiversion = float(xbmcaddon.Addon('xbmc.addon')..getAddonInfo('version')[0:4])
```

**Pull request merged**